frontend/components/simulation/SimulationLogs.stories.js

```diff
@@ -0,0 +1,34 @@
+import { storiesOf } from '@storybook/vue';
+import faker from 'faker';
+import SimulationLogs from './SimulationLogs';
+
+const randomLogs = length => {
+    return faker.datatype.array(length).map(() => {
+        return `${faker.date.recent().toISOString()}: ${faker.random.words(3)}`;
```

Comment:
Can you please use `sentence` instead, to see how it handles long, multi-line log messages?

Reply:
Screenshot below 🙂